website/app/templates/about.hbs

```diff
@@ -20,8 +20,8 @@
     </p>
 
     <h2 class="doc-text-h2">Contributions</h2>
-    <p class="doc-text-body">As an open-source project, we think a lot about how people can contribute—and there are
-      quite a few ways our consumers are encouraged to participate! Read our
+    <p class="doc-text-body">As an open source project, we think a lot about how people can contribute—and there are
+      quite a few ways our consumers are encouraged to participate. Read our
       <a class="doc-link-generic" href="/about/contribution/" target="_blank" rel="noopener noreferrer">contribution</a>
       documentation to find the different ways anyone can contribute to Helios.
```

Comment:
```suggestion
      documentation to find the different ways someone can contribute to Helios.
```

Reply:
Either is ok but I'm not sure we should imply just "anyone" can contribute.

Reply:
Could also change the "can" to "could" in here to further soften a bit the implication that others can contribute in the way normally understood in "open source" projects.

Reply:
added these in, thanks!